### ISSUE TITLE
Final Fantasy XIII series cannot load to intro after #5677 

### ISSUE THREAD

**Issue by kding99** [RPCS3/rpcs3#5812]
Final Fantasy XIII series cannot load to intro after #5677 
it always shows the following error

E {PPU[0x1000000] Thread (main_thread) [0x014cff0c]} '_sys_prx_get_module_id_by_name' failed with 0x8001112e : CELL_PRX_ERROR_UNKNOWN_MODULE [2]
E {RSX [0x1ff0b600]} RSX: FIFO error: possible desync event x10357

rpcs3-v0.0.6-7827-e38b7aee_win64
CPU-Intel I7-2600
RAM-DDR3-1333Mhz
GPU-RX 570 4Gb
[RPCS3.log.zip](https://github.com/RPCS3/rpcs3/files/3061620/RPCS3.log.zip)

**Comment by elad335** [comment]
Game is trying to map rsx memory on vm::main, very odd.

**Comment by elad335** [comment]
@kding99 When you got time post log with [this](https://ci.appveyor.com/project/elad335/rpcs3/builds/23731242) build.

**Comment by Asinin3** [comment]
Here's a log from xiii
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/3064318/RPCS3.log.gz)

**Comment by elad335** [comment]
Fixed by #5792

**Comment by ikki84** [comment]
@elad335 @Asinin3 E {RSX [0x1ff0b600]} RSX: FIFO error: possible desync event , still persists in the recent rpcs3 master version. Could you test it again?

**Comment by loreeffe** [comment]
> 
> 
> @elad335 @Asinin3 E {RSX [0x1ff0b600]} RSX: FIFO error: possible desync event , still persists in the recent rpcs3 master version. Could you test it again?

FIFO error: possible desync event , still persists in #5988 v0.0.6-8147 Alpha [2019-05-19]

**Comment by elad335** [comment]
No need to update if nothing changed, and there's a pull request which already fixing it (the mgs4 fixes).

**Comment by Hopefullyidontgetbanned** [comment]
Fixed

**Comment by loreeffe** [comment]
Yes great progress guys after #6112 it now boots with Vulkan but not OpenGL, colors of characters and lights are wrong, WCB option does not solve this at least on my system (i5 4440 8Gb RAM Nvidia 710). Save game issue still persist.

**Comment by AniLeo** [comment]
This issue is fixed with #6112, so closing
For other regressions, you may want to open a new issue
